### ISSUE TITLE
Increase the contrast for the hover state of the Toolbar Icon Button

### ISSUE THREAD

**Issue by myvuuu** [Workday/canvas-kit#2884]
## 💡 Suggestion

Update the hover state of the Toolbar Icon button to meet a11y color contrast requirements. 

### Success Criteria

- All states of Toolbar Icon Button meets contrast requirements
- Toolbar Icon Button utilizes the new set of css variable tokens

## Motivation

The background contrast is not being met for the Toolbar Icon Button's hover state. Additionally, changing from grey to blue is color alone makes it difficult for users with CVD to view.

## Example

Please provide some examples of this suggestion in practice.

**Comment by williamjstanton** [comment]
Suggesting under Motivation: 
To improve conformance with WCAG 2.2 guidelines [1.4.1 Use of color](https://www.w3.org/WAI/WCAG22/Understanding/use-of-color.html) and [1.4.11 Non-text contrast](https://www.w3.org/WAI/WCAG22/Understanding/non-text-contrast.html). 

Toggle buttons with an inverted color scheme:
<img width="542" alt="jira format toolbar" src="https://github.com/user-attachments/assets/c9cd5931-3e61-42de-82e1-3155cedbbe66">

Toggle buttons with an outline:
<img width="901" alt="outlook format toolbar" src="https://github.com/user-attachments/assets/19ea3128-ee3d-4518-86fb-cfc394157b95">

Toggle buttons with a fill or no-fill scheme: 
<img width="95" alt="slack-fill-no-fill" src="https://github.com/user-attachments/assets/115e6fb3-5d7b-473f-8e0e-9b23a6f9cc1e">

**Comment by aek5ba** [comment]
Hi @williamjstanton - just wanted to check back in to see if this is something that you'd still like to see completed? And is this currently an accessibility violation, or is this an enhancement? Thanks!

**Comment by williamjstanton** [comment]
> Hi [@williamjstanton](https://github.com/williamjstanton) - just wanted to check back in to see if this is something that you'd still like to see completed? And is this currently an accessibility violation, or is this an enhancement? Thanks!

Yes, I would. The background color change signaling the 2 states of the button does not satisfy [1.4.11 Non-text contrast](https://www.w3.org/WAI/WCAG22/Understanding/non-text-contrast.html). It's hard for users to see and understand which state the toggle is in.
